src/UKHO.S100PermitService.API/Program.cs

```diff
@@ -113,12 +115,14 @@ private static void ConfigureServices(WebApplicationBuilder builder)
             builder.Services.AddApplicationInsightsTelemetry(options);
             builder.Services.AddControllers();
             builder.Services.AddEndpointsApiExplorer();
+            builder.Services.AddMemoryCache();
             builder.Services.AddDistributedMemoryCache();
 
             builder.Services.Configure<EventHubLoggingConfiguration>(configuration.GetSection(EventHubLoggingConfiguration));
             builder.Services.Configure<HoldingsServiceApiConfiguration>(configuration.GetSection(HoldingsServiceApiConfiguration));
             builder.Services.Configure<UserPermitServiceApiConfiguration>(configuration.GetSection(UserPermitServiceApiConfiguration));
             builder.Services.Configure<ProductKeyServiceApiConfiguration>(configuration.GetSection(ProductKeyServiceApiConfiguration));
+            builder.Services.Configure<ManufacturerKeyConfiguration>(configuration.GetSection(ManufacturerKeyvault));
```

Comment:
configuration name should be ManufacturerKeyVault instead of ManufacturerKeyvault.

Reply:
Done